courses/cunix/ex02/src/my_strcmp.c

```diff
@@ -0,0 +1,18 @@
+int my_strcmp(char *str1, char *str2){
+	unsigned int sum1 = 0, sum2 = 0;
+	while(*str1 != '\0'){
+		sum1 += *str1;
+		str1++;
+	}
+	while(*str2 != '\0'){
+		sum2 += *str2;
+		str2++;
+	}
+	if(sum1 == sum2){
+		return 0;
+	}else if(sum1 > sum2){
+		return 1;
+	}else{
+		return -1;
+	}
```

Comment:
this doesn't look like the best solution, lots of repeatable code that should be avoided
0.75